packages/analytics/src/analytics.ts

```diff
@@ -15,6 +15,7 @@ export class Analytics {
     lastname?: string;
     email?: string;
     created?: Date;
+    body?: Record<any, any>;
```

Comment:
I think we can improve this better by using this type for params:

```typescript
createAccount(params: {
  accountId: string;
  firstname?: string;
  lastname?: string;
  email?: string;
  created?: Date;
  [key: string]: any;
}) {
  // ...
}
```

This is so that `body` is no longer needed.